### DISCUSSION TITLE
请问更换了内核就不能使用TUN模式吗？

### DISCUSSION THREAD

**Discussion by laozi4vip** [MerlinKodo/clash-rev#12] [Q&A]
请问更换了内核就不能使用TUN模式吗？ 有办法解决吗？  谢谢

**Comment by MerlinKodo** [comment]
CFW还是clash verge, CFW换了之后要在管理员身份下运行tun模式才会生效

**Reply by laozi4vip** [reply]
CFW，我用管理员模式打开clash还是没用呢

**Reply by MerlinKodo** [reply]
![image](https://github.com/MerlinKodo/clash-rev/assets/150133030/9919b7d2-e4c0-4c54-a4b6-7bd39f6c3ee2)
我正在用是可以正常走tun的。。
